src/Microsoft.AspNetCore.Razor/TagHelpers/DefaultTagHelperContent.cs

```diff
@@ -282,11 +277,10 @@ private bool IsEmptyOrWhiteSpaceCore(object entry, EmptyOrWhiteSpaceWriter write
         {
             if (entry == null)
             {
-                return false;
+                return true;
```

Comment:
This is the only real change. Everything else done to this file was VS suggestions.